### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by AlexSpaceBy**
Homework task 2

### CODE REVIEW

File: hometask2.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+func median(i []int) float64{
+
+	sort.Ints(i)
+
+	if len(i) % 2 != 0 {
+
+		return float64(i[len(i)/2])
+	}
+
+	var mid = (float64(i[len(i)/2-1]) + float64(i[len(i)/2]))/2
```
Comment (VolhaVetrava) [inline hometask2.go:23]: var mid is redundant, just write `return float64(i[len(i)/2-1]) + float64(i[len(i)/2]))/2`

File: hometask2.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+type Square struct {
+	start Point
+	a     uint
+}
+
+func (s Square) End() string{
+
+	x := strconv.Itoa(s.start.x + int(s.a))
+	y := strconv.Itoa(s.start.y + int(s.a))
```
Comment (VolhaVetrava) [inline hometask2.go:41]: First of all, basic math - it should be `y - int(s.a)`

Next, why are you using strconv.Itoa?
This is much simpler and more logical:
`return Point{s.start.x + int(s.a), s.start.y - int(s.a)}`

File: hometask2.go (hunk 1 above)
Comment (AlexSpaceBy) [reply hometask2.go:23]: Will be fixed in the following commit.

File: hometask2.go (hunk 2 above)
Comment (AlexSpaceBy) [reply hometask2.go:41]: strconv.Itoa helps to create output in a more convenient way: (x; y). 
Will be fixed in the following commit.

**Pull request closed without merging**